Generate docs (#2356)
kyma-project/cli: docs/user/gen-docs/01-60-kyma_alpha_hana.md

# kyma alpha hana

> Manage a Hana instance on the Kyma platform

Reviewer: ```suggestion
Manages an SAP HANA instance on SAP BTP.
```

Author: chaning to the `on a Kyma cluster`

## Synopsis

> Use this command to manage a Hana instance on the SAP Kyma platform

Reviewer: ```suggestion
Use this command to manage an SAP HANA instance on SAP BTP.
```

Author: changing to the `on a Kyma cluster`

## Flags inherited from parent commands

```bash
  -h, --help                    Help for the command
      --kubeconfig string       Path to the Kyma kubeconfig file
      --show-extensions-error   Print possible error when fetching extensions failed
```

## See also

> * [kyma alpha](kyma_alpha.md)                   - Groups command prototypes the API for which may still change

Reviewer: ```suggestion
* [kyma alpha](kyma_alpha.md)                   - Groups command prototypes for which the API may still change.
```

> * [kyma alpha hana map](kyma_alpha_hana_map.md) - Map the Hana instance to the Kyma cluster

Reviewer: ```suggestion
* [kyma alpha hana map](kyma_alpha_hana_map.md) - Maps an SAP HANA instance to the Kyma cluster.
```

# kyma alpha hana

> Manages an SAP HANA instance on the Kyma cluster.

Author: ```suggestion
Manages an SAP HANA instance in the Kyma cluster.
```

## Synopsis

> Use this command to manage an SAP HANA instance on the Kyma cluster.

Author: ```suggestion
Use this command to manage your SAP HANA instance in the Kyma cluster.
```
